e2e/ansible/pre-check.yml

```diff
@@ -0,0 +1,55 @@
+- hosts: localhost
+
+  vars:
+    list: ["default", "kube-public", "kube-system", "openebs"]
+
+  tasks:
+
+    - name: Cleanup /var/openebs
```

Comment:
Can we use file module here to delete the directory?

Reply:
Modified task using file module.